article/test/ArticleAmpValidityTest.scala

```diff
@@ -4,6 +4,11 @@ import org.scalatest.DoNotDiscover
 
 
 @DoNotDiscover class ArticleAmpValidityTest extends AmpValidityTest {
+
+  /*
+      NB: these may be duplicated in tools/amp-validation - if you add or remove
+      a URL from here, think about whether it should be removed there too.
+   */
```

Comment:
Can we consolidate all those urls in a single location? Like a file both tests would read to get all the urls.
I am not sure developers read comments 😜